src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -112,7 +160,7 @@ func Init(version string) (*Config, error) {
 						return cfg.updateFrom(bytes)
 					}
 				} else {
-					log.Errorf("Nil bytes?")
+					log.Debugf("bytes is nil when the config is unmodified")
```

Comment:
I think we can completely remove this line or L422, as it always follows after the "Config unchanged in cloud" message.